tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Dashboard/PublishedStatus.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+
+test.describe('Published Status Dashboard tests', () => {
+
+  test.beforeEach(async ({umbracoUi}) => {
+    await umbracoUi.goToBackOffice();
+    await umbracoUi.publishedStatus.goToSection(ConstantHelper.sections.settings);
+    await umbracoUi.publishedStatus.clickPublishedStatusTab();
+  });
+
+  test('can view and refresh published cache status', async ({umbracoApi, umbracoUi}) => {
```

Comment:
You can create a content item, and check if the ContentStore contains the content or not.